cardano-node/app/cardano-cli.hs

```diff
@@ -290,6 +292,21 @@ parseTxRelatedValues =
                   "Path to signing key file, for genesis UTxO using by generator."
       ]
 
+parseMiscCommands :: Parser ClientCommand
```

Comment:
Should the README also be updated?

Reply:
@Jimbo4350, this code moves out to a different repo -- and there will be docs etc.